apps/application-system/form/src/routes/AssignApplication.tsx

```diff
@@ -41,7 +42,7 @@ export const AssignApplication = () => {
     },
   )
 
-  useEffect(() => {
+  useEffectOnce(() => {
```

Comment:
What is the reason for this change?

Reply:
When we were reviewing the state changes and history logs we noticed that when the employer opens the email notifying them of the employer's parental leave the state transition gets triggered twice (because of useEffect - strict in React 18). This set the whole history log chain out of sync. We are aware that this isn't the case on prod but wanted to be able to see it correctly on dev.